tests/helpers.js

```diff
@@ -104,6 +104,24 @@ export function validLangpackManifestJSON(extra) {
   }, extra));
 }
 
+
+export function validStaticThemeManifestJSON(extra) {
+  return JSON.stringify(Object.assign({}, {
+    manifest_version: 2,
+    name: 'My Static Theme',
+    version: '1.0',
+    theme: {
+      images: {
+        headerURL: 'weta.png',
+      },
+      colors: {
+        accentcolor: '#adb09f',
+        textcolor: '#000',
```

Comment:
Maybe add some extra `xxxcolor` to test the other styles of colour?  Any valid css colour property should work - in particular you need `rgb(r g, b, a)` style to do partial transparency.

Reply:
I'll add some more tests.

Reply:
:+1: I also agree